docs/source/fabric_definition.rst

```diff
@@ -600,15 +600,15 @@ FABulous defines the following coding rules for BELs:
     
     TODO 
   
-  * ``SHARED_PORT``: this directive can only be used together optionally with ``EXTERNAL``. If a port is set ``EXTERNAL`` but not ``SHARED_PORT``, then , a TODO ( shared  ports flagged with this directive are not connected to the switch matrix but are exported through the tile entity to the top-level fabric wrapper.
+  * ``SHARED_PORT``: this directive can only be used together optionally with ``EXTERNAL``. If a port is set ``EXTERNAL`` but not ``SHARED_PORT``, then, a TODO ( shared  ports flagged with this directive are not connected to the switch matrix but are exported through the tile entity to the top-level fabric wrapper.
```

Comment:
This is a weirdly inlined TODO anyway

```suggestion
  * ``SHARED_PORT``: this directive can only be used together optionally with ``EXTERNAL``
  
TODO - shared  ports flagged with this directive are not connected to the switch matrix but are exported through the tile entity to the top-level fabric wrapper.
```

Reply:
I was also thinking about using the [Sphinx builtin TODO](https://www.sphinx-doc.org/en/master/usage/extensions/todo.html#directive-todolist). Maybe this would be a better option?

Reply:
I would say so!

Reply:
I will use it then!

Reply:
After using the todo extenstion, it looks like this:

![image](https://github.com/user-attachments/assets/815f2477-d721-4d88-b055-e7d968ae119f)

Should the to-do be included in the text at all? Or should we just use the todo extension to have the todos in the source files but not in the actual documentation? If we want to have it in the actual documentation, we should then also create a separate page for the to-dos. I personally would just keep it inside the source and hide it in the docs since it looks to much like work in progress if you see a to-do somewhere. But this is of course debatable :)

Reply:
Oh, maybe I misunderstood - do you mean Sphinx has an option to hide the TODOs in the actual builds? That also works

Reply:
> Hmm yeah, this is a little in your face. Maybe the best option is actually just to comment it out (especially since the VPR flow might end up deprecated anyway)

We have three other TODOs (all in `fabric_definition.rst`). I would apply the same method to all of them to keep it consitent.

> Oh, maybe I misunderstood - do you mean Sphinx has an option to hide the TODOs in the actual builds? That also works

The TODOs are actually hidden by default (see also this [SO discussuion](https://stackoverflow.com/questions/50314856/enabling-the-todo-feature-in-sphinx-docs)).  There is also the `todo_emit_warnings` option which should show warnings when building, but it is not yet working for me. This would IMHO be the best approach since TODOs can easily be forgotten over time ;)

Reply:
Ah, ok, great! Agreed then, if the TODO blocks aren't shown by default then wrapping things up in them sounds good! 

I would hold off on `todo_emit_warnings` for now as the ReadTheDocs build is configured to fail on warnings, so this will cause all our docs to stop building. Could always be something to change later and modify the RTD config though.

Reply:
I did manage to make the warnings appear. I did not change the source file after adding `todo_emit_warnings` and therefore the file was not rebuilt..
`make clean` to the rescue :) 

> [...] the ReadTheDocs build is configured to fail on warnings, so this will cause all our docs to stop building [...]

I assumed that something like this would be the case. Then I will leave it for now of course.